plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -369,6 +369,16 @@ func buildControllerRoles() ([]rbacv1.ClusterRole, []rbacv1.ClusterRoleBinding)
 			eventsRule(),
 		},
 	})
+	addControllerRole(&controllerRoles, &controllerRoleBindings, rbacv1.ClusterRole{
```

Comment:
this added an unconditional role / binding to default cluster policy

this should be wrapped in a feature gate check so we don't create these permissions until the feature is enabled

fixed in #121682

Reply:
thanks for fixing it